src/test/java/unrc/dose/UserProfileTest.java

```diff
@@ -0,0 +1,72 @@
+package unrc.dose;
+
+import static org.junit.Assert.*;
+
+import org.javalite.activejdbc.Base;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class UserProfileTest {
+	
+	private User newUser = User.set("Enzo" , "Ferrari" , "F40@gmail.com" , false);
+	
+	@BeforeClass
+	
+  	public static void beforeAll() {
+  		if (!Base.hasConnection()) {
+	      	Base.open();
+	      	Base.openTransaction(); 
+	    }
+  	}
+	
+	@AfterClass
+  	public static void tearDown() {
+    	Base.rollbackTransaction();
+    	Base.close();
+	}
+
+	/**
+	 * Test that proves that display name and twitter id of a given user are correct
+	 */
+	@Test
+	public void testSetGetCorrect() {
+		
+		UserProfile up = UserProfile.createUserProfile(newUser.getId(),"Enzo","@Enzo");
+		assertEquals(true,up.getDisplayName());
+		assertEquals(true,up.getTwitterId());
```

Comment:
The types are incorrect, the getter methods from `UserProfile` return a `String`